src/views/Books/BookDetail.js

```diff
@@ -1,18 +1,26 @@
 import { useEffect, useState } from 'react';
 import Book from '../../components/book/Book';
 import { getBookById } from '../../services/books';
+import { useParams } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 
 function BookDetail() {
-  const id = 1; // TODO: Use id from route
+  const params = useParams(); // TODO: Use id from route
+
   const [book, setBook] = useState(null);
 
   useEffect(() => {
-    getBookById(id).then(({ data }) => setBook(data));
-  }, [id]);
+    getBookById(params.id).then(({ data }) => setBook(data));
+  }, [params.id]);
 
   if (!book) return <h3>Loading book...</h3>;
 
-  return <Book book={book} showDetail />;
+  return (
+    <>
+      <Link to={`/books`}>Back to Catalog</Link>
```

Comment:
For the Link tag you can just have the path in strings, example: <Link to="/books">